instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/exporters/logging/SystemOutLogRecordExporterAutoConfiguration.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.spring.autoconfigure.exporters.logging;
+
+import io.opentelemetry.exporter.logging.SystemOutLogRecordExporter;
+import io.opentelemetry.instrumentation.spring.autoconfigure.OpenTelemetryAutoConfiguration;
+import io.opentelemetry.instrumentation.spring.autoconfigure.exporters.internal.ExporterConfigEvaluator;
+import org.springframework.boot.autoconfigure.AutoConfigureBefore;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Condition;
+import org.springframework.context.annotation.Conditional;
+import org.springframework.context.annotation.Configuration;
+
+/** Configures {@link SystemOutLogRecordExporter} bean for tracing. */
+@Configuration
+@AutoConfigureBefore(OpenTelemetryAutoConfiguration.class)
+@Conditional(SystemOutLogRecordExporterAutoConfiguration.CustomCondition.class)
+@ConditionalOnClass(SystemOutLogRecordExporter.class)
+public class SystemOutLogRecordExporterAutoConfiguration {
+
+  @Bean(destroyMethod = "") // SDK components are shutdown from the OpenTelemetry instance
+  @ConditionalOnMissingBean
+  public SystemOutLogRecordExporter otelSystemOutLogRecordExporter() {
+    return SystemOutLogRecordExporter.create();
+  }
+
+  static final class CustomCondition implements Condition {
+    @Override
+    public boolean matches(
+        org.springframework.context.annotation.ConditionContext context,
```

Comment:
any reason why these couldn't be imported?

Reply:
No, replaced by import